R/hypothesis_apply.R

```diff
@@ -47,6 +47,16 @@ hypothesis_functions <- list(
                 out[!is.na(out)]
             })
     ),
+    trt_vs_ctrl = list(
+        ratio = list(
+            comparison = function(x) mean(x[2:length(x)] / x[1]),
```

Comment:
I actually think this example is too specific to be very useful - what if I have 2 or even 3 control groups (also very common in psych, with sham-control, passive-control, etc...)